code/src/index.css

```diff
@@ -1,13 +1,189 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: #ffe5bb;
+  /* fallback for old browsers */
+  background: -webkit-linear-gradient(
+    45.34deg,
+    #119da4 5.66%,
+    #9bc7b2 31.53%,
+    #ffe5bb 50.01%,
+    #ffbe99 72.18%,
+    #ff886a 94.35%
+  );
+  /* Chrome 10-25, Safari 5.1-6 */
+  background: linear-gradient(
+    45.34deg,
+    #119da4 5.66%,
+    #9bc7b2 31.53%,
+    #ffe5bb 50.01%,
+    #ffbe99 72.18%,
+    #ff886a 94.35%
+  );
+  /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+header {
+  max-height: 20%;
+  font-family: "Roboto";
+  color: rgb(17, 157, 164);
+  text-align: center;
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+}
+
+.super-pic {
+  width: 300px;
+  height: 300px;
+  border: 1px solid whitesmoke;
+  border-radius: 50%;
+  position: absolute;
+  top: 8%;
+  left: 2%;
+}
+
+form {
+  min-height: 100vh;
+  max-width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  align-items: center;
+}
+
+.question-card {
+  width: 450px;
+  height: 450px;
+  height: min-content;
+  border: 2px solid black;
+  border-radius: 15px;
+  padding: 20px;
+  box-shadow: 10px 10px 42px 0px rgba(0, 0, 0, 0.75);
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  text-align: center;
+  margin: 5px;
+}
+
+label {
+  color: darkorchid;
+  font-size: 17px;
+}
+
+.question-header {
+  color: rgb(17, 157, 164);
+  width: 100%;
+  position: relative;
+}
+
+.checkbox-input {
+  width: fit-content;
+  height: 20%;
+  color: aqua;
+}
+
+.summary-card {
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 20px;
+}
+
+button {
+  width: 75px;
+  height: 75px;
+  display: inline-block;
+  font-size: 16px;
+  background-color: rgb(103, 226, 253);
+  border-radius: 50%;
+  cursor: pointer;
+}
+
+button:hover {
+  background-color: rgb(255, 136, 106);
+}
+
+.summary-text {
+  color: rgb(17, 157, 164);
+  font-size: 20px;
+  font-weight: bolder;
+}
+
+.footer {
+  text-align: center;
+}
+
+.footer p {
+  color: rgb(255, 212, 172);
+  font-weight: bold;
+}
+
+/* MEDIA QUERIES */
+
+@media only screen and (max-width: 500px) {
+  .question-card {
+    width: 300px;
+    height: 100px;
+  }
+
+  h1 {
+    display: none;
+  }
+
+  .super-pic {
+    left: 2%;
+  }
+}
+
+/* Small devices (landscape phones, less than 768px) */
+@media only screen and (max-width: 767.98px) {
```

Comment:
The superwoman image and third question options in mobile view would need  to be adjusted a bit for a fully reponsive experience. But this was not a requirement for this weeks project.
![image](https://user-images.githubusercontent.com/47081254/97580721-f9957100-19f3-11eb-9f06-2cefc74a5a28.png)